deck/templates/event/snippets/event_list_item.html

```diff
@@ -19,8 +19,25 @@ <h3 class="panel-title event-title">{{ event.title }}</h3>
             <i class="icon-exclamation-sign"></i>
             {% trans "Due date pasted since the day" %} {{ event.due_date }}
           </span>
+          {% if event.event_date %}
+            <span>
+              {{ event.event_date }}
+            </span>
+          {% endif %}
         {% else %}
-          {% trans "Due date in" %} <span class="{% if event.due_date_is_close %}text-danger{% else %}text-warning{% endif %}">{{ event.due_date|timeuntil }}</span>
+          <span>
+            {% blocktrans with event_title=event.title %}
+              The event: <strong>{{ event_title }}</strong>
+            {% endblocktrans %}
+            {% if event.event_date %}
```

Comment:
I think that the `event_date` is required, isn't?

Reply:
It is not. It is an optional field